build.gradle

```diff
@@ -46,6 +46,7 @@ spotless {
   scala {
     target 'streams/**/*.scala'
     scalafmt("$versions.scalafmt").configFile('checkstyle/.scalafmt.conf')
+    licenseHeaderFile 'checkstyle/java.header', 'package'
```

Comment:
This will enforce nothing like this happens again by applying the same check we have for Java via checkstyle.

Reply:
One correction, we still don't check python/markdown/gradle/etc. files, so don't let your guard down!

Still, we're now covered for java and scala.